tests/security/attributes/run_test.pl

```diff
@@ -32,8 +32,9 @@
 my $sub_expect = "0";
 my $pub_timeout = "10";
 my $sub_timeout = "10";
+my $pub_extra_space = "0";
 
-GetOptions ( 'scenario=s' => \$scenario, 'pub_cfg=s' => \$pub_cfg_file, 'sub_cfg=s' => \$sub_cfg_file, 'pub_cert=s' => \$pub_cert_file, 'sub_cert=s' => \$sub_cert_file, 'pub_key=s' => \$pub_key_file, 'sub_key=s' => \$sub_key_file, 'gov=s' => \@gov_files, 'pub_perm=s' => \@pub_perm_files, 'sub_perm=s' => \@sub_perm_files, 'topic=s' => \@topic_names, 'pub_expect=i' => \$pub_expect, 'sub_expect=i' => \$sub_expect, 'pub_timeout=i' => \$pub_timeout, 'sub_timeout=i' => \$sub_timeout );
+GetOptions ( 'scenario=s' => \$scenario, 'pub_cfg=s' => \$pub_cfg_file, 'sub_cfg=s' => \$sub_cfg_file, 'pub_cert=s' => \$pub_cert_file, 'sub_cert=s' => \$sub_cert_file, 'pub_key=s' => \$pub_key_file, 'sub_key=s' => \$sub_key_file, 'gov=s' => \@gov_files, 'pub_perm=s' => \@pub_perm_files, 'sub_perm=s' => \@sub_perm_files, 'topic=s' => \@topic_names, 'pub_expect=i' => \$pub_expect, 'sub_expect=i' => \$sub_expect, 'pub_timeout=i' => \$pub_timeout, 'sub_timeout=i' => \$sub_timeout , 'pub_extra_space=i' => \$pub_extra_space);
```

Comment:
I know this was like this before, but this makes it easier to read and edit.

```suggestion
GetOptions(
  'scenario=s' => \$scenario,
  'pub_cfg=s' => \$pub_cfg_file,
  'sub_cfg=s' => \$sub_cfg_file,
  'pub_cert=s' => \$pub_cert_file,
  'sub_cert=s' => \$sub_cert_file,
  'pub_key=s' => \$pub_key_file,
  'sub_key=s' => \$sub_key_file,
  'gov=s' => \@gov_files,
  'pub_perm=s' => \@pub_perm_files,
  'sub_perm=s' => \@sub_perm_files,
  'topic=s' => \@topic_names,
  'pub_expect=i' => \$pub_expect,
  'sub_expect=i' => \$sub_expect,
  'pub_timeout=i' => \$pub_timeout,
  'sub_timeout=i' => \$sub_timeout,
  'pub_extra_space=i' => \$pub_extra_space);
```

Reply:
Right, we have a guideline for this.  In general if you're modifying a line/function/file and it doesn't follow a guideline, go ahead and update it.

https://github.com/objectcomputing/OpenDDS/blob/master/docs/guidelines.md#text-file-formatting